src/test/ui/async-await/mutually-recursive-async-impl-trait-type.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0733]: recursion in an `async fn` requires boxing
+  --> $DIR/mutually-recursive-async-impl-trait-type.rs:5:18
+   |
+LL | async fn rec_1() {
+   |                  ^ recursive `async fn`
```

Comment:
It'd be nice if we could point out where the recursion is coming from, but that can be left for a follow up diagnostics improvement.